images/win/scripts/Installers/Install-Toolset.ps1

```diff
@@ -48,7 +48,7 @@ $ErrorActionPreference = "Stop"
 Import-Module -Name ImageHelpers -Force
 
 # Get toolcache content from toolset
-$ToolsToInstall = @("Python", "Node")
+$ToolsToInstall = @("Python", "Node", "Go")

```

Comment:
I believe we should also change `go.sh` script?

Reply:
Should we setup default version in toolset as for python or stay on separate go script for installation ?

Reply:
Done.